### DISCUSSION TITLE
Twitter Görevleri

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#885] [Soru Cevap - Türkçe]
Twitter görevleri yapmadığımız zaman kayboluyormu son güne bırakmak mantıklımı bilen varsa lütfen  cevap versin.

**Comment by Skips1** [comment]
Şu ana kadar Hiç kaybolmadı kanka ama kaybolmayacağı anlamına gelmez bu yüzden işini sona bırakma derim iyi günler

**Reply by ghost** [reply]
Teşşekür ederim
